pnlscripts/epi.py

```diff
@@ -86,7 +86,7 @@ def main(self):
             unu('convert', '-t', 'uchar', '-i', epimask, '-o', epimask)
 
             if '.nhdr' in dwiepi.suffixes:
-                unu("save", "-e", "gzip", "-f", "nrrd", "-i", dwiepi, self.out)
+                unu("save", "-e", "gzip", "-f", "nrrd", "-i", dwiepi, "-o", self.out)
```

Comment:
Hi @alfredomp , good catch, thanks for contributing. Before we merge, can you do one test?
It looks like the error happens only when user wants to save as `.nhdr` and should not happen when saving as `.nrrd`. Can you verify the latter?

Reply:
Actually, I had an image lying somewhere and I have been able to verify that for us. Thanks!

Reply:
Hi @tashrifbillah, I just saw your messages. Thank you for testing. 
Happy to contribute whenever possible.